### DISCUSSION TITLE
HI about this Speedy.exe

### DISCUSSION THREAD

**Discussion by KISS-MY-WEED** [community/community#35948] [Other Feature Feedback, Questions, & Ideas]
I got a question |
Is this normal? 
29 out of 72 antivirus say is bad file?!?
![Screenshot (605)](https://user-images.githubusercontent.com/115663423/195431690-fb422cad-b04b-44f0-b75a-eb7c5d72a4c6.png)

**Comment by radar-blip** [comment]
It doesn't seem like something thats normal. 😄

**Reply by KISS-MY-WEED** [reply]
I got some more files to test out what it say like this 1:
Warframe-Undetected-cheat-with-Spoofer-main ``Speedy.exe`` again just a bit bigger file size 9KB VS 160KB.
And got same bad result?!?  0-0 
The SITE: https://www.virustotal.com
![Screenshot (608)](https://user-images.githubusercontent.com/115663423/195445421-8b5477ae-bb37-4ecf-88d2-0fad673e1a3b.png)

**Reply by Whiskey992** [reply]
Gone fishing 🎣

**Reply by KISS-MY-WEED** [reply]
Have u played Warframe? If not is not PVP and 1month ago DE made some very annoying ammo reduction. So i just wanna test the ammo function. But 47 of 72 say better don't run file. And i am asking you pros for some help ? To figure out is this normal?

**Comment by nooksmr** [comment]
Hi.

Maybe because the software injects something or looks at bad spaces. Or maybe because it's a virus.
No cheats, no security issues, no problems.

King regards,
